lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -43,6 +43,13 @@ def define
         option :yes
         option :no
 
+        precalculate :exempt_benefits do
+          exempt_benefits = query.data.fetch("exempt_benefits").map do |exempt_benefit|
+            "- #{exempt_benefit}"
+          end
+          exempt_benefits.join("\n")
+        end
```

Comment:
This is a view concern and so should really be in the receiving_exemption_benefits.govspeak.erb template.